build.sbt

```diff
@@ -21,8 +21,8 @@ def specs2(scalaVersion: String) =
     ("org.specs2" %% s"specs2-$n" % "4.14.1").cross(CrossVersion.for3Use2_13) % Test
   }
 
-val jacksonVersion         = "2.11.4"
-val jacksonDatabindVersion = jacksonVersion
+val jacksonVersion         = "2.13.2"
+val jacksonDatabindVersion = "2.13.2.2"
```

Comment:
Only databind has that `.2` release, core and others don't have that, you can check here:
* https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/
* https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-core/